packages/daemon/src/mail.js

```diff
@@ -60,11 +60,32 @@ export const makeMailboxMaker = ({
       );
     };
 
-    const terminate = async petName => {
-      const formulaIdentifier = lookupFormulaIdentifierForName(petName);
```

Comment:
i would expect `lookupFormulaIdentifierForNamePath` or just `lookupFormulaIdentifierForName` to support paths and not require any dance in implementation land

Reply:
As part of introducing petname paths, the design decision was that only the naming hub `lookup()` would be made variadic.

Currently, "lookup" is an overloaded word in the codebase. @kriskowal and I discussed renaming pet store-associated methods that include the word "lookup" to use some other word, like "locate", instead. Would that help?